### DISCUSSION TITLE
Alternatives For Picom

### DISCUSSION THREAD

**Discussion by car8675309** [i3/i3#5819] [Q&A] [answered]
Sorry if this is a duplicate, I'm not used to using GitHub discussions instead of reddit and didn't see anything about this topic in my searches.

I decided to go back to using i3 with picom as the compositor. It seems that picom keeps causing my memory usage to be at 100% and causing my computer to crash (I am assuming there is a memory leak). Does anyone have any other suggestions for a compositor?

**Comment by orestisfl** [comment]
You'll probably have better luck asking in the picom discussions: https://github.com/yshui/picom/discussions

Otherwise, I'll try changing the backend option and disabling everything in the config and enable options one by one until you find what causes the issue.

How long does it take for the problem to manifest?

**Reply by car8675309** [reply]
Thanks so much for pointing me there. 

I will also try that too

It takes anywhere from 5-10min

**Reply by car8675309** [reply, marked as answer]
Looks like it has to do with having `blur-background` being set to `true`. Thanks for the advice!
